### ISSUE TITLE
Let's migrate to main branch

### ISSUE THREAD

**Issue by willingc** [jupyter/accessibility#39]
After #35 merges let's move the default branch to main.

**Comment by bollwyvl** [comment]
No reason to hold up on #35! I've updated that to only run its workflows when targeting the `ci` branch to avoid burning lots of coal for the (hoped-for) upcoming docs work, so when the new HEAD is dubbed, I would request `ci` also gets created, and I'll update the target.

No doubt there would be some on-going back-and-forth between `main` and `ci`, as we'd definitely want the binder on `main` to be "good," showing the current state of demo-ready software, but also reflect our best efforts on improving them, while at times `ci` might be left "broken" (e.g. failing CI).

Right now, the workflows only fail if the _test infrastructure_ fails (e.g. `502` from registry.npmjs.org/anconda.org, `ENOENT too many files open` from webpack, etc.), but the pa11y return codes are currently discarded because of the 1000s of web content accessibility findings. But once we establish a [roadmap](https://github.com/pydata/pydata-sphinx-theme/pull/294/files#diff-5fa26e114dbffb75fdeaf7c7d81d832210b1b4e88972b0fab6afcedd1191d4e9), un-expected failures (or happy unexpected successes!), I feel there _should_ be :heavy_multiplication_x: if there are regressions, or we can celebrate and remove a previously-skipped rule.

**Comment by tonyfast** [comment]
who has permissions on this repo to change the default main branch name? @isabela-pf is that something you can do?

**Comment by isabela-pf** [comment]
I checked and I do not have the power! Is this a permission over the repo, or the whole Jupyter org?

**Comment by jasongrout** [comment]
I just gave the accessibility team admin permission on this repo, so you should have permission to do it now.

**Comment by trallard** [comment]
Hey folks can we please migrate to `main` this has been on hold for quite a while

**Comment by trallard** [comment]
This has been done by @tonyfast  and @isabela-pf 🎉  thank you 🙏🏽
